### PR TITLE
fix: Update readme to make code example run

### PR DESCRIPTION
**Pull request by cazeth**
Hi, had to to make a couple updates to the code example in the readme to get it to run on my machine. Might have been something I did wrong, but this runs out of the box for me. Sharing in case it's useful.

### CODE REVIEW

**Comment by malik672** [comment]
ohh, make sense tbh

File: README.md [hunk 1]
```diff
@@ -58,7 +59,8 @@ fn main() {
     println!("Are the tokens equal? {}", dai_token.equals(&another_dai_token));
 
     // Example of sorting tokens
-    let another_token = token!(CHAIN_ID, "0x0000000000000000000000000000000000000002", DECIMALS, "ETH", "Ethereum");
+    const ANOTHER_TOKEN_ADDRESS: &str = "0x0000000000000000000000000000000000000002";
+    let another_token = token!(CHAIN_ID, ANOTHER_TOKEN_ADDRESS, DECIMALS, "ETH", "Ethereum");
```
Comment (shuhuiluo) [inline README.md]:
```suggestion
    let another_token = token!(
        CHAIN_ID,
        "0000000000000000000000000000000000000002",
        DECIMALS,
        "ETH",
        "Ethereum"
    );
```
`token!` can take either string literal or `&str`. If passing string literal to `address!` macro, "0x" should be stripped.

File: README.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ the `token!` macro.
 // having to import each dependency individually.
 // Import necessary preludes and types
 use uniswap_sdk_core::prelude::*;
+use uniswap_sdk_core::token;
```
Comment (shuhuiluo) [inline README.md]:
```suggestion
use crate::{prelude::*, token};
```

**Review by shuhuiluo — CHANGES_REQUESTED** [review]

**Review by shuhuiluo — APPROVED** [review]
Thx.

**Pull request merged**